Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            //Get input from the user.
+            string fName = GetStringInput("Enter your first name: ");
```

Comment:
No need to use abriaviation in naming variables, unless they are common knowledege